src/tests/Nightwatch/Tests/AddContent.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  '@tags': ['AddContent'],
+  addContentRenders(browser) {
+    browser
+      .logUserIn()
+      .relativeURL('/node/add')
+      .waitForElementVisible('[data-nightwatch="node-type-list"] a', 10000)
```

Comment:
Instead for specifying 10000 for every single of those links I'm wondering whether we should change the default instead, see https://github.com/jsdrupal/drupal-admin-ui/pull/214